db/schema.rb

```diff
@@ -20,28 +19,28 @@
     t.string "encrypted_password", default: "", null: false
     t.integer "failed_attempts", default: 0, null: false
     t.string "unlock_token"
-    t.datetime "locked_at"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "locked_at", precision: nil
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
```

Comment:
The churn of these fields is a result in a MySQL fix datetime columns in scheme files. Somewhere in the Rails changelog they explain this.

Reply:
Ah, cool. This, correct?

<img width="1038" alt="image" src="https://github.com/indyhackers/indyhackers.org/assets/2425/556d4fc7-49a7-4b1a-af52-3eb2098549e3">